app/models/ems_refresh/save_inventory_networks.rb

```diff
@@ -153,12 +153,12 @@ def save_network_routers_inventory(ems, hashes, target = nil)
       store_ids_for_new_records(ems.network_routers, hashes, :ems_ref)
     end
 
-    def save_network_ports_inventory(ems, hashes, target = nil)
+    def save_network_ports_inventory(ems, hashes, target = nil, source = :neutron)
```

Comment:
If one `:source` is `"smartstate"`, then I think the other source should be `"inventory"`.   That feels more consistent to me.